clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -1003,6 +1003,9 @@ void invokeCompletedOffsetCommitCallbacks() {
         RequestFuture<Void> future =  null;
```

Comment:
Not related to you change but there is an extra space before `null`. Do you mind fixing this in your PR?

Reply:
Done.